### ISSUE TITLE
Purchases: Non-refundable domains can display as refundable

### ISSUE THREAD

**Issue by drewblaisdell** [Automattic/wp-calypso#7588]
**Reproducing**
You need an account with a domain that is beyond the refund period with auto-renew disabled.
- Visit `/domains/add/:site` for the site with the aforementioned domain.
- Purchase a domain for this site.
- Visit `/purchases` and click on the domain that is beyond the refund period.
- Assert that you are given the option to both remove or receive a refund for this domain:

<img width="730" alt="screen shot 2016-08-22 at 11 00 29 am" src="https://cloud.githubusercontent.com/assets/1130674/17859537/02fe7fae-6858-11e6-8926-7f200f317765.png">

**Comment by yurynix** [comment]
I was unable to reproduce that, the steps I've done:

In the sandboxed store:
1. I've registered a domain.
![screen shot 2016-08-30 at 1 51 12 pm](https://cloud.githubusercontent.com/assets/326402/18086843/95e32f46-6eba-11e6-97e4-162cffdeece8.png)

2. I bumped it's transaction 2 month ahead in the backend
3. I've disabled auto renew in SA.

The result is:
![screen shot 2016-08-30 at 2 03 08 pm](https://cloud.githubusercontent.com/assets/326402/18086861/af44abe0-6eba-11e6-968e-23c34eb40e4a.png)
